### ISSUE TITLE
Share extra attributes

### ISSUE THREAD

**Issue by PVince81** [owncloud/ocis-reva#350]
It's the field "attributes" in OCS Share.

It's currently used in OC 10 for the additional permissions related to secure view.

**Comment by PVince81** [comment]
@pmaier1 FYI probably comes much later

**Comment by pmaier1** [comment]
Yep, thanks for opening this. This should in the future be handled with configurable roles in sharing, I think. Will become relevant for online office integrations.
